ConcernsCaseWork/ConcernsCaseWork.CypressTests/cypress/support/commands.js

```diff
@@ -391,15 +391,7 @@ Cypress.Commands.add('validateCaseManagPage', () => {
         expect($row.eq(3).text().trim()).to.contain('Concerns').and.to.match(concernsRgx).and.to.match(/(Add concern)/).and.to.match(/(Edit)/);
     })
 
-    cy.get('[class^="govuk-accordion__section govuk-accordion"]').should(($accord) => {
```

Comment:
Should we be changing the test to check these values are on the page still? (Or is that elsewhere in the test)?
They've just moved rather than being removed.

Reply:
Yeah, I'm just checking that you are deliberately removing tests rather than fixing them.

Reply:
I have looked into this with @mikestock-nimble and we are both in agreement to remove this as we do not see the value in this test as it only checks if edit is there when we should really be clicking on it instead.

I have added this to our list when we are refactoring the create, view and edit case tests for Cypress.

Reply:
Cool - guess that's covered in https://dfe-gov-uk.visualstudio.com/Academies-and-Free-Schools-SIP/_workitems/edit/86478 and https://dfe-gov-uk.visualstudio.com/Academies-and-Free-Schools-SIP/_workitems/edit/86481. Also that the edit button is present only when the current user is the 'created by' user.